typings/index.d.ts

```diff
@@ -0,0 +1,93 @@
+declare module 'reduxful' {
+  // fetchUtils
+  export interface RequestAdapterOptions {
+    method: string;
+    url: string;
+    headers: { [key: string]: string };
+    withCredentials: boolean;
+    body: any;
+  }
+
+  export type RequestAdapter = (options: RequestAdapterOptions) => Promise<any>;
+
+  export type Fetcher = (url: string, outOpts: Object) => Promise<any>;
```

Comment:
I believe that the DOM library in TypeScript has types defined for `fetch`. That'd be an improvement over `Object` here.

Reply:
Yeah, if you have the `dom` lib specified you can use the syntax below. It's possible there may be a more appropriate interface defined somewhere, but I didn't see one in the `dom` lib.

```suggestion
  export type Fetcher = typeof fetch;
```